beater/server_test.go

```diff
@@ -274,7 +301,7 @@ func randomAddr() string {
 	return l.Addr().String()
 }
 
-func waitForServer(secure bool, host string) {
+func waitForServer(secure bool, host string, timeout int) {
```

Comment:
would prefer a context to timeout here but we can add that later

Reply:
Do you mean the name of the variable is not intelligible ?